playbooks/robusta_playbooks/job_actions.py

```diff
@@ -125,4 +125,108 @@ def job_events_enricher(event: JobEvent, params: EventEnricherParams):
         event.add_enrichment([events_table_block], {SlackAnnotations.ATTACHMENT: True})
 
 
+class JobPodEnricherParams(EventEnricherParams):
+    """
+    :var events: Add the events of the related pod
+    :var logs: Add the logs of the related pod
+    """
+    events: bool = True
+    logs: bool = True
+
+
+@action
+def job_pod_enricher(event: JobEvent, params: JobPodEnricherParams):
+    """
+    Given a Kubernetes job, get information about the latest job pod.
+
+    Used to get the related pod's events and/or logs
+    """
+    job = event.get_job()
+    if not job:
+        logging.error(
+            f"cannot run job_pod_enricher on alert with no job object: {event}"
+        )
+        return
+
+    pod = get_job_latest_pod(job)
+
+    if pod:
+        if params.events:
+            events_table_block = get_resource_events_table(
+                "*Job pod events:*",
+                pod.kind,
+                pod.metadata.name,
+                pod.metadata.namespace,
+                included_types=params.included_types,
+                max_events=params.max_events,
+            )
+            if events_table_block:
+                event.add_enrichment([events_table_block], {SlackAnnotations.ATTACHMENT: True})
+
+        if params.logs:
+            log_data = pod.get_logs()
```

Comment:
if there are multiple containers in the job will this give us all of the logs or just or the first container?
(like if the initContainer causes the event or if there is multiple containers)
Have you tested this on jobs with multiple containers? (I dont know if its common since I havn't ever needed to used jobs)

Reply:
it gives us the logs of the first container.
This is the way we do it everywhere with the logs_enricher.
It's not perfect, but I think it's ok